src/psc_flatfoil_yz.cxx

```diff
@@ -213,8 +224,12 @@ void setupParameters()
   g.target_n = 2.5;
   g.target_Te = 0.001;
   g.target_Ti = 0.001;
+  
+  g.electron_HE_ratio = 0.01;
+  
   g.target_Te_heat = 0.04;
-  g.target_Ti_heat = 0.04;
+  g.target_Ti_heat = 0.0;
```

Comment:
was `target.Ti_heat` ever used, by the way (I'd think probably not, but if it was, then we might have to revisit this, as it means I misremembered something).

Reply:
good catch, it should have been I think.  otherwise `fac[ion] != 0 `,  added the change